src/toolkit/Community.VisualStudio.Toolkit.Shared/Solution/SolutionItem.cs

```diff
@@ -73,6 +83,11 @@ internal void Update(IVsHierarchyItem item)
         /// </summary>
         public IEnumerable<SolutionItem?> Children => _children ??= _item.Children.Select(t => FromHierarchyItem(t));
 
+        /// <summary>
+        /// Returns whether the item is normally visible in soltuion explorer
```

Comment:
@StevenRasmussen 1) typo `soltuion`; 2) also I guess the term `normally` can raise  questions for the programmer, may be add more context here? :)

Reply:
I would change "visible" to "hidden". Maybe something like this:

> Returns whether the item is normally hidden in solution explorer and only visible when Show All Files is enabled.

Reply:
Agreed. Thanks!